### PR TITLE
Hello microverse project

### PR DESCRIPTION
**Pull request by Epaltechs**
In this project, I:
- added a .gitignore file
- added a HTML file
- added a CSS file
- added a test.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,12 +9,6 @@ My first project during the first week of training in Microverse.
 - CSS
 - Linters
 ​
-​
-​
-## Getting Started
```
Comment (Deepakdanger) [inline README.md:14]: - [ ] Let's add this section also, which contains the necessary code for running the project locally. For ex:-
```
git clone https://github.com/Epaltechs/Hello-World.git
cd Hello-World
```

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @Epaltechs, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please consider while commit:-
- Commit should be in the present tense.
- Commit should start with a Capital letter. For ex:- ` Add index.html file`.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Epaltechs** [comment]
In this commit, I:
- Add the getting started section in the Readme.md file

File: README.md [hunk 2]
```diff
@@ -9,11 +9,10 @@ My first project during the first week of training in Microverse.
...
 ​
-​
-​
 ## Getting Started
 ​
-To get a local copy up and running follow these simple example steps.
+git clone https://github.com/Epaltechs/Hello-World.git
+cd Hello-World
 ​
 ## Authors
```
Comment (ShoiraTa) [inline README.md:24]: - [ ] OPTIONAL: I noticed that the Authors section is empty, please consider adding your social links in Authors section eg: 

```
## Author
👤 **Your_name**

- GitHub: [@gihub_name](githib_url)
- LinkedIn: [linkedin_name](linkedin_url)

![Epaltechs's GitHub stats](https://github-readme-stats.vercel.app/api?username=epaltechs&count_private=true&theme=dark&show_icons=true)

```

File: README.md [hunk 3]
```diff
@@ -9,11 +9,10 @@ My first project during the first week of training in Microverse.
 - CSS
 - Linters
 ​
-​
-​
 ## Getting Started
 ​
-To get a local copy up and running follow these simple example steps.
+git clone https://github.com/Epaltechs/Hello-World.git
+cd Hello-World
```
Comment (ShoiraTa) [inline README.md]: - [ ] OPTIONAL: You have done a good job adding how to clone the repository, but currently it lacks information and requires proper layout. Try this: 
```
## Getting Started

To get a local copy up and running follow these simple example steps.

- copy this link `https://github.com/Epaltechs/Hello-World.git`.
- get the directory that you want to clone the repository.
- open the command prompt in this directory.
- write `git clone `https://github.com/Epaltechs/Hello-World.git`.
- go to the repository folder in your command prompt `cd Hello-World`

```

**Review by ShoiraTa — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Epaltechs,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

parks-and-rec-great-job-gif

## Optional suggestions

_Check the comments below_

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

***
Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (Epaltechs) [reply README.md]: I appreciate it so much.

**Pull request merged**